pulsar_spectra/catalogue_papers/Bates_2011.yaml

```diff
@@ -94,7 +94,7 @@
    9.0
   ],
   "Flux Density error mJy": [
-   0.2
```

Comment:
This and the following value are incorrect. The 9(2) uncertainty format means that the value in the parenthesis has the same decimal places as the last number so it would be +/-2. In the next instance 12(2) it would also be +/2

Reply:
I am confused because, in the original, you had shown 9 with an error of 0.2. I changed it to 2; for 12(2), you had shown 0.02, which I also changed to 2. Did I make any mistakes in my modifications?

Reply:
No, you are correct! Sorry, I got confused by diff description